app/src/main/java/studyBuddy/timemanagement/SessionRecord.java

```diff
@@ -52,4 +52,8 @@ public static SessionRecord fromString(String input) {
         Gson gson = new Gson();
         return gson.fromJson(input, SessionRecord.class);
     }
+
```

Comment:
we probably just need an additional thing here so that we can record the actual duration vs expected